recipes/pyreadline3/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "pyreadline3" %}
+{% set version = "3.3" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pyreadline3-{{ version }}.tar.gz
+  sha256: ff3b5a1ac0010d0967869f723e687d42cabc7dccf33b14934c92aa5168d260b3
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: True  # [not win]
```

This looks like it can be `noarch: python`:

```suggestion
  noarch: python
```

---

This one is a Windows only package. You cannot even install it on a different platform.

https://github.com/brgirgis/pyreadline3/blob/46725bb4aee97a7a0dc85cc55ed93a16e22f40d1/setup.py#L19-L21

---

Oops, I hastily read it as `skip: True  # [win]` 😅 Sorry about that!